thirdeye/thirdeye-frontend/app/pods/custom/metrics-table-changes/wo1w/template.hbs

```diff
@@ -1,3 +1,6 @@
-<span class="rootcause-metric__table--{{record.wo1w.direction}}">
+<p class="metrics-table__baseline-score">
```

Comment:
BLOCKER: Any reasons we built 4 different templates here? 
It feels like this could be accomplished with the same template and just pass 1. a score and 2. a value?

Reply:
that's a legacy hack of mine. if you find a better solution please go ahead, but it wasn't immediately clear how pass separate key/value pairs for the same "record"